watson/watson.py

```diff
@@ -581,3 +581,9 @@ def rename_tag(self, old_tag, new_tag):
 
         self.frames.changed = True
         self.save()
+    
+    def style(self = None, name = 'error', element = None):
+      if isinstance(self, Watson):
```

Comment:
This is an hack, I should have probably commented it in the code. Here is the rationale.

Having `self` default to `None` is a way to have both (sort of) a class method (used in case the object has not yet been instantiated, for instance by exceptions in `cli`) and an instance method.

If called as `Watson.style` this will not attempt to look at the `config` property of the uninstantiated object; on the other hand, if `watson` is an instance of `Watson`, then `watson.style` will be able to access the configured formats.